examples/benchmarks/ollama.rs

```diff
@@ -20,7 +20,7 @@ macro_rules! result_format_str {
 #[tokio::main]
 async fn main() {
     let models = ["orca-mini"]; //, "phi3", "llama3", "openhermes"];
-    let preset_prompts =  [  
+    let preset_prompts = [
```

Comment:
interesting enough, this two-char leading-space was not auto-fixable by the formatter and required manual change on my end as well hahah